dojo/models.py

```diff
@@ -3053,9 +3056,15 @@ def set_sla_expiration_date(self):
         if not system_settings.enable_finding_sla:
             return
 
+        # some parsers provide date as a `str` instead of a `date` in which case we need to parse it #12299 on GitHub
+        sla_start_date = self.get_sla_start_date()
+        from dateutil.parser import parse
```

Comment:
This should be at the top of the file

Reply:
Good point. I assumed it was like this to avoid circular references/imports. Changed it.